src/mono/msbuild/apple/data/ProxyProjectForAOTOnHelix.proj

```diff
@@ -49,14 +50,14 @@
 
   <Target Name="BundleTestAppleApp" DependsOnTargets="AppleBuild" />
 
-  <Target Name="_PrepareForAppleBuildAppOnHelix" DependsOnTargets="_PublishRuntimePack">
+  <Target Name="_PrepareForAppleBuildAppOnHelix">
```

Comment:
Revisiting this, we moved _PublishRuntimePack to `AppleBuildDependsOn` to make the order of targets more clear. 

Just wondering if this target `_PrepareForAppleBuildAppOnHelix` directly depends on `_PublishRuntimePack` (if so I think we should keep the DependsOnTargets here in addition to adding it to AppleBuildDependsOn).

Are there other targets or parts of the build that depends on `_PublishRuntimePack`? If not, then I think we should keep it clear that we only have `_PublishRuntimePack` for `_PrepareForAppleBuildAppOnHelix` and remove it from `AppleBuildDependsOn`

Reply:
I think it's preference. I like the `AppleBuildDependsOn` because it's one place to see all the targets that run.

Reply:
That makes sense, I was thinking that if we only needed `_PublishRuntimePack` for whenever `_PrepareForAppleBuildAppOnHelix` is ran, then we ensure that `_PublishRuntimePack` will only be ran in tandem with `_PrepareForAppleBuildAppOnHelix` (maybe same condition), otherwise over time it might be difficult to simplify. 

e.g. if at some point `_PrepareForAppleBuildAppOnHelix` is no longer needed and is removed, but `_PublishRuntimePack` is kept because it wasn't clear that `_PublishRuntimePack` target was only included to ensure `_PrepareForAppleBuildAppOnHelix` would work properly